client/src/components/Footer.jsx

```diff
@@ -1,19 +1,7 @@
-
-const Footer = () => 
-   (
-    <footer  
-    style={{
-      position: 'fixed',
-      bottom: 0,
-      left: 0,
-      width: '100%',
-      justifyContent: 'center',
-      textAlign: 'center',
-    }}
-    >
-      <p>&copy; 2023 - Kandoo</p>
-    </footer>
-  );
-;
+const Footer = () => (
+  <footer className="fixed bottom-0 left-0 w-full flex justify-center items-center">
+    <p>&copy; 2023 - Kandoo</p>
+  </footer>
```

Comment:
Any edge cases we need to be aware of? 

AND

can you make it so that the year changes per current year? So that it isn't static?

Reply:
Maybe responsive. I don't know what else edge cases could be